import.go

```diff
@@ -31,7 +31,7 @@ func resolveOCIIndex(idx ocispec.Index, refObject string) (*ocispec.Descriptor,
 			return &m, nil
 		}
 	}
-	return nil, errors.Errorf("not found: %q", refObject)
+	return nil, errors.Errorf("not found:m %q", refObject)
```

Comment:
is this a stray `m` or does it mean something? 😇

Reply:
dang, yes it is. Probably a failed emacs command :sweat_smile: 

Thanks for noticing